### DISCUSSION TITLE
New icons to add

### DISCUSSION THREAD

**Discussion by caplationut** [tabler/tabler-icons#142] [Ideas]
HI!
In the 1250 icons I did not find one with a dove. Can you make one and add it to the package? I would need it for my project.
Thank you!

**Comment by KODISHALA** [comment]
Thanks for creating such nice looking icons. I am looking for 'Patient' and 'Doctor' icons. Can you please make and add these to your package. Thanks in advance.

**Reply by codecalm** [reply]
check it: https://github.com/tabler/tabler-icons/releases/tag/v1.44.0 :)
